crypto/initthread.c

```diff
@@ -38,6 +38,13 @@ struct thread_event_handler_st {
 #ifndef FIPS_MODULE
     const void *index;
 #endif
+    /*
+     * prio is an arbitrary uint32_t defining the order in which
+     * cleanup handlers are run.  Higher values get run later than
+     * lower values.  Handlers with the same priority are run in
+     * an unspecified order
+     */
+    uint32_t prio;
```

Comment:
I think `prio` is an unfortunate name here. Typically in a thread context, we use `prio` as run/execution priority. The member is supposed to define a clean up order so `order` (or something similar) feels like more natural choice for me.